packages/modules/src/modules/banners/supporterMoment/SupporterMomentBanner.stories.tsx

```diff
@@ -0,0 +1,132 @@
+                textColour: brand[400],
+                border: `1px solid ${brand[400]}`,
+            },
+            hover: {
+                backgroundColour: '#FDF1F8',
+                textColour: brand[400],
+            },
+            theme: 'brand',
+        },
+        highlightedTextSettings: {
+            textColour: 'white',
+            highlightColour: '#721765',
+        },
+        choiceCards: true,
+        bannerId: 'supporter-moment-banner',
+    }),
+    'supporter-moment-banner',
+);
+
+const SupporterMomentTemplate: Story<BannerProps> = (props: BannerProps) => (
+    <SupporterMomentBanner {...props} />
+);
+
+export const SupporterMoment = SupporterMomentTemplate.bind({});
+SupporterMoment.args = {
+    ...props,
+    content: {
+        heading: 'The only pocket we’re in is yours',
+        messageText:
+            'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, semi. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo.',
+        paragraphs: [
+            'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, semi. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. ',
+        ],
+        highlightedText:
+            'Nullam dictum felis eu pede mollis pretium. Integeir tincidunt. Thank you.',
+        cta: {
+            text: 'Contribute',
+            baseUrl: 'https://support.theguardian.com/contribute/one-off',
+        },
+    },
+    mobileContent: {
+        heading: 'The only pocket we’re in is yours',
+        messageText:
+            'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, semi. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo.',
+        paragraphs: [
+            'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, semi. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. ',
+        ],
+        highlightedText:
+            'Nullam dictum felis eu pede mollis pretium. Integeir tincidunt. Thank you.',
+        cta: {
+            text: 'Contribute',
+            baseUrl: 'https://support.theguardian.com/contribute/one-off',
+        },
+    },
+    choiceCardAmounts: {
+        testName: 'Storybook_europeLocalLanguage',
```

Comment:
This doesn't need to be unique for storybook, it's typically copied over and in most other places it has a generic name. I'd suggest keeping the generic name for ease
```suggestion
        testName: 'Storybook_test',
```

Reply:
Yep, all done